DuneBase/Dockerfile

```diff
@@ -15,6 +15,8 @@ ENV C_COMPILER=${ccompiler}
 ARG cppcompiler "g++-12"
 ENV CPP_COMPILER=${cppcompiler}
 
+ENV DUNE_CMAKE_FLAGS=-DCMAKE_C_COMPILER=${ccompiler}\ -DCMAKE_CXX_COMPILER=${cppcompiler}
```

Comment:
Ah maybe the python bindings ignore the flags passed below at line 41?

Can you then move all flag passed below also up here to `DUNE_CMAKE_FLAGS`? Because then it is not doubled below.

Reply:
Yeah so from my understanding from https://gitlab.dune-project.org/core/dune-common/-/issues/237
the JIT compiler doens't know about the flags passed in at line 41, so we have to set both flags. 

So I should add `-DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=20` etc. to `DUNE_CMAKE_FLAGS`